class/defaults.yml

```diff
@@ -169,7 +173,6 @@ parameters:
       dataDirHostPath: /var/lib/rook
       monitoring:
         enabled: ${rook_ceph:ceph_cluster:monitoring_enabled}
-        rulesNamespace: ${rook_ceph:ceph_cluster:namespace}
```

Comment:
Had to be removed or Argo fights with the operator. The operator tries to remove it.

Reply:
This parameter doesn't exist in Rook 1.9, since the operator doesn't manage the Prometheus alert rules anymore, cf. https://github.com/rook/rook/releases/tag/v1.9.0

However, the old operator-managed rules were not deleted as far as I can tell, we should probably add a post-sync job to delete the PrometheusRule object `prometheus-ceph-v16-rules` in ${rook_ceph:ceph_cluster:namespace}`.